model/stacktrace_frame.go

```diff
@@ -43,6 +44,9 @@ func (s *StacktraceFrame) Transform(config *pr.Config, service Service) common.M
 	enhancer.Add(m, "module", s.Module)
 	enhancer.Add(m, "function", s.Function)
 	enhancer.Add(m, "vars", s.Vars)
+	if config != nil && config.LibraryPattern != nil && s.LibraryFrame == nil {
+		s.LibraryFrame = s.isLibraryFrame(config.LibraryPattern)
```

Comment:
`isLibraryFrame` can never return `nil`, so it should return a `bool` instead of `*bool` and then we should store it in a variable here and set `s.LibraryFrame` to the address of that var.